scripts/minikube.sh

```diff
@@ -50,16 +50,28 @@ function detect_minikube() {
 
 # install minikube
 function install_minikube() {
+     local mku_version latest_version
+     mku_version=$(${minikube} update-check 2> /dev/null || true)
+     latest_version=$(echo ${mku_version} | cut -d' ' -f4)
```

Comment:
I have an old version of minikube, that outputs:
```
$ minikube update-check
CurrentVersion: v0.27.0
LatestVersion: v1.12.3
```
Better set `latest_version` to `awk '/LatestVersion {print $NF}' <<< "${mku_version}"` so that the order of the lines, or additions do not influence the result.

This should also probably be inside the `if type ${minikube}` statement in case minikube is not installed yet.

Reply:
Indention looks a little off too, compared to the original lines below.

Reply:
> I have an old version of minikube, that outputs:
> 
> ```
> $ minikube update-check
> CurrentVersion: v0.27.0
> LatestVersion: v1.12.3
> ```
> 
> Better set `latest_version` to `awk '/LatestVersion {print $NF}' <<< "${mku_version}"` so that the order of the lines, or additions do not influence the result.

This won't work Niels, as storing the output of the command will already convert the above multiline output to single line.

✨ mku_version=$(minikube update-check 2> /dev/null || true)
✨ echo $mku_version
CurrentVersion: v1.12.3 LatestVersion: v1.12.3

But I get your point, will defend on it.

> 
> This should also probably be inside the `if type ${minikube}` statement in case minikube is not installed yet.

I'm trying to capture the latest_version which will be used after the if block, and absence of minkube command will be taken care, as it will result in empty string ${mku_version}

Reply:
> Indention looks a little off too, compared to the original lines below.

Hmm, tweaked my vimrc, thanks!